include/erlcloud_ddb2.hrl

```diff
@@ -447,3 +447,9 @@
        {table_description :: undefined | #ddb2_table_description{}
        }).
 -endif.
+
+-record(ddb_request,
+       {headers :: erlcloud_ddb_impl:headers(),
```

Comment:
Should `headers` be exported from `_impl`, rather than from `_ddb2`?

Reply:
I checked on whether there was any type called `headers` on `_ddb2` however, I only found the type specification on `_impl` thus exported from there.

Reply:
I think I would actually delete that type from the `_impl` module and put it at the top of `erlcloud_ddb2.hrl` with the other types there (`date_time()`, etc.).